modules/efk-logging-elasticsearch-exposing.adoc

```diff
@@ -40,28 +62,57 @@ metadata:
   name: elasticsearch
   namespace: openshift-logging
 spec:
-  host:
+  host: 
   to:
     kind: Service
     name: elasticsearch
   tls:
-    termination: reencrypt <1>
-    destinationCACertificate: |-   <2>
-      -----BEGIN CERTIFICATE-----
-      [...]
-      -----END CERTIFICATE-----
+    termination: reencrypt
+    destinationCACertificate: <1>
+----
+<1> Add the Elasticsearch CA ceritifcate or use the command in the next step. You do not need to set the `spec.tls.key`, `spec.tls.certificate` and `spec.tls.caCertificate` parameters
+required by some reencrypt routes.
+
+.. Run the following command to add the Elasticsearch CA certificate to the route YAML you created:
++
+----
+cat ./admin-ca | sed -e "s/^/      /" >> expose-es.yaml
```

Comment:
Since we call the yaml file <file-name>.yaml at the line 85 "oc create -f <file-name>.yaml", we'd better use <file-name>.yaml here, too.
`$ cat ./admin-ca | sed -e "s/^/      /" >> <file-name>.yaml`